kotlin/display-grid/src/main/java/com/esri/arcgisruntime/sample/displaygrid/MainActivity.kt

```diff
@@ -60,6 +62,7 @@ class MainActivity : AppCompatActivity() {
     mapView.apply {
       // create a map with imagery basemap
       map = ArcGISMap(Basemap.createImagery()).also { map ->
```

Comment:
how about for readability, moving the `mapview.grid` (which could be just `grid`) up to below the `map = ArcGISMap` code line. Not sure about the pattern of nesting the `also` within the `apply` since I've been so used to keeping the two separate, but I don't think there's a problem with readability as long as the `grid` logic is moved up.

Reply:
I'm a bit confused here...do you mean to move it directly below where it would be inside the also block? I'll remove the also altogether